README.md

```diff
@@ -23,88 +28,133 @@ gem 'mongoid-locker'
 and run `bundle install`. In the model you wish to lock, include `Mongoid::Locker` after `Mongoid::Document`. For example:
 
 ```ruby
-class QueueItem
+class User
   include Mongoid::Document
   include Mongoid::Locker
 
+  field :locking_name, type: String
   field :locked_at, type: Time
-  field :locked_until, type: Time
 
-  field :completed_at, type: Time
+  field :age, type: Integer
 end
 ```
 
 Then, execute any code you like in a block like so:
 
 ```ruby
-queue_item.with_lock do
-
-  # do stuff
-
-  queue_item.completed_at = Time.now.utc
-  queue_item.save!
+user.with_lock do
+  user.age = 17
+  user.save!
 end
 ```
 
-The `#with_lock` function takes an optional [handful of options around retrying](http://rdoc.info/github/mongoid/mongoid-locker/Mongoid/Locker:with_lock), so make sure to take a look.
-
-The default timeout can also be set on a per-class basis:
-
-```ruby
-class QueueItem
-  # ...
-  timeout_lock_after 10
-end
-```
+The `#with_lock` function takes an optional [handful of options](https://www.rubydoc.info/gems/mongoid-locker/Mongoid/Locker#with_lock-instance_method), so make sure to take a look.
```

Comment:
Minor, we should remove these links, as they point to some latest version of mongoid-locker and should probably be just the list below.